modules/module-extras.php

```diff
@@ -16,6 +16,7 @@
 	'theme-tools/social-menu.php',
 	'theme-tools/content-options.php',
 	'custom-post-types/comics.php',
+	'custom-post-types/redirector.php',
```

Comment:
Placed here, the tool is only `init`ed when Jetpack has a connection or it's in development mode. I'm pretty sure this is the correct behavior here, but we can also include it below to run whenever the plugin is active (regardless of "connected" status). 

@marekhrabe @jeherve do you have opinions here?

Reply:
This should be fine imo.